core/src/main/scala/zio/logging/AbstractLogging.scala

```diff
@@ -2,24 +2,20 @@ package zio.logging
 
 import zio.{ Cause, ZIO }
 
-trait AbstractLogging[Message] {
-  def logging: AbstractLogging.Service[Any, Message]
+trait AbstractLogging {
```

Comment:
Why were all the type parameters deleted? LogStage specifically requires a non-String message type.

Reply:
instead of param here you can change ```LogFormat``` and one part of application can use different ```LogFormat```s